EventGenerator/src/GammaConvGenerator_module.cc

```diff
@@ -0,0 +1,153 @@
+// Generates particles in a flat shaped spectrum, size determined by FCL params these will be attached to a mu- in
+// the input SimParticleCollection.
+// This module throws an exception if no suitable muon is found.
+//
+// S Middleton, 2021
+
+#include <iostream>
+#include <string>
+#include <cmath>
+#include <memory>
+
+#include "CLHEP/Vector/ThreeVector.h"
+#include "CLHEP/Vector/LorentzVector.h"
+#include "CLHEP/Random/RandomEngine.h"
+#include "CLHEP/Random/RandFlat.h"
+#include "CLHEP/Units/PhysicalConstants.h"
+
+#include "fhiclcpp/types/Atom.h"
+#include "messagefacility/MessageLogger/MessageLogger.h"
+
+#include "art/Framework/Core/EDProducer.h"
+#include "art/Framework/Principal/Event.h"
+#include "art/Framework/Services/Registry/ServiceHandle.h"
+#include "art_root_io/TFileService.h"
+#include "Offline/SeedService/inc/SeedService.hh"
+#include "Offline/GlobalConstantsService/inc/GlobalConstantsHandle.hh"
+#include "Offline/GlobalConstantsService/inc/ParticleDataList.hh"
+#include "Offline/GlobalConstantsService/inc/PhysicsParams.hh"
```

Comment:
Is this include used?  Could you check what of these are actually used?

Reply:
cleaned